### PR TITLE
My first Pull Request

### PR DESCRIPTION
**Pull request by ElenaRiabova**
Stage 0. Git та GitHub (6)

### CODE REVIEW

**Comment by suchov** [comment]
It looks like too many changes @ElenaRiabova maybe your IDE configuration, can you check it in the "Files changed" folder?

**Comment by ElenaRiabova** [comment]
Can you please explain more about what I should do in the tab "Files changed"?
I made 4 commits because I got message that my file wasn't up to date

**Comment by suchov** [comment]
Yeh, you have pretty much the same issue as in this PR https://github.com/kottans/mock-repo/pull/1014
Hope the answers in it will help, LMK if it's still confusing.

File: README.md [hunk 1]
```diff
@@ -32,11 +32,13 @@ Good luck to all of you!
 
 
 # Hello GitHub
+
 It's my first time, so please be gentle, Git))
 
-#  ,,,^._.^,,,
+# ,,,^.\_.^,,,
```
Comment (suchov) [inline README.md:38]: remove extra lines editing

File: README.md [hunk 2]
```diff
@@ -45,28 +47,30 @@ This is [link](https://github.com/Dovahkiin1991/kottans-frontend) to my repo.
 [![Kottans-Git](https://img.shields.io/badge/%3D%28%5E.%5E%29%3D-git-orange.svg)](https://github.com/kottans/git-course)
 
 # Hi, kottans ! I`m Max
-thanks for the awesome course!
 
+thanks for the awesome course!
```
Comment (suchov) [inline README.md:51]: like this one too

**Review by suchov — CHANGES_REQUESTED** [review]

**Comment by ElenaRiabova** [comment]
> Yeh, you have pretty much the same issue as in this PR #1014 Hope the answers in it will help, LMK if it's still confusing.

I see all the changes but I can't edit them. I don't get the massage about merge conflict or see them in my file

**Comment by ElenaRiabova** [comment]
It's getting out fo control, I'm closing this PR and starting anew.
Obviously, I don't have enough experience to fix/reverse/delete my mistakes now, maybe later I'll understand what went wrong

**Pull request closed without merging**